ipam/api.go

```diff
@@ -18,7 +18,7 @@ var (
 	errAddressPoolNotFound     = fmt.Errorf("Address pool not found")
 	errAddressPoolInUse        = fmt.Errorf("Address pool already in use")
 	errAddressPoolNotInUse     = fmt.Errorf("Address pool not in use")
-	errNoAvailableAddressPools = fmt.Errorf("No available address pools")
+	ErrNoAvailableAddressPools = fmt.Errorf("No available address pools")
```

Comment:
it's easier to find them when you put exported vars in a separate block

Reply:
because they might be wrapped or appear in logs, error strings should not be sentence case (and should not have closing punctuation)

Reply:
Do I change all other error strings to lower case, too? They're not part of this work item: 14337706.

Reply:
😕
You did the right thing and matched the convention here. But it could be something to file as a future repair, and keep in mind when creating new error strings later.